src/components/BobaContainer/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import bobalist from './bobalist.jsx';
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import './index.css'
+
+const BobaContainer = () => {
+  return (
+    <div className='indBoba'>
+        {bobalist.map((boba) => (
+            <Card key={boba.name} style={{ width: '18rem' }}>
+                <Card.Img variant="top" src="holder.js/100px180" />
+                <Card.Body>
+                    <Card.Title>{boba.name}</Card.Title>
```

Comment:
Suggest destructuring const {name, price} = boba instead of repetitive `boba.name`,  `boba.price`